.github/workflows/test-package-downloads-action.yml

```diff
@@ -461,7 +461,7 @@ jobs:
       - name: Download Onedir Tarball as an Artifact
         uses: actions/download-artifact@v3
         with:
-          name: ${{ inputs.package-name }}-${{ inputs.salt-version }}-onedir-darwin-${{ matrix.arch }}.tar.xz
+          name: ${{ inputs.package-name }}-${{ inputs.salt-version }}-onedir-darwin-${{ matrix.arch == 'arm64' && 'aarch64' || matrix.arch }}.tar.xz
```

Comment:
Wondering about the '&&' for am64 and aarch64,

Reply:
if arch is `arm64` replace it with `aarch64` otherwise, leave it alone.

What's your doubt here?